frontend/src/components/course/Links.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="links">
+        <p>
+          <a target=_blank :href="handbookURL">Handbook <i class="material-icons">launch</i></a>
+        </p>
+        <p v-if="outlineURL">
+          <a target=_blank :href="outlineURL">Course Outline <i class="material-icons">launch</i></a>
+        </p>
+    </div>
+</template>
+
+<script>
+export default {
+  props: {
+    handbookURL: String,
+    outlineURL: String
+  }
+}
+</script>
+
+<style scoped>
+.links {
+    margin-top: 20px;
+    align-items: center;
+    display: flex;
+    border-bottom: var(--border);
```

Comment:
idk about this... but leave it in for now

Reply:
Yeah also not sure. Temporary. Discuss tomorrow. One thing was maybe metrics should sit in space on right. Dunno.